bitbots_vision/src/bitbots_vision/vision.py

```diff
@@ -232,6 +232,17 @@ def _configure_vision(self, config, level):
                     config,
                     self._package_path)
 
+        # Check if params changed
+        if ros_utils.config_param_change(self._config, config,
+                r'^green_color_detector_|field_color_detector_use_hsv'):
+            # Deactivate dynamic color space
```

Comment:
As this is a duplicate of the code above, I suggest outsourcing this.

Reply:
Which part is duplicated? The parameters in the are different in the both cases.